src/Map/assets/dist/leaflet_controller.js

```diff
@@ -0,0 +1,110 @@
+import { Controller } from '@hotwired/stimulus';
+import 'leaflet/dist/leaflet.min.css';
+import L from 'leaflet';
+
+/******************************************************************************
+Copyright (c) Microsoft Corporation.
+
+Permission to use, copy, modify, and/or distribute this software for any
+purpose with or without fee is hereby granted.
+
+THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES WITH
+REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF MERCHANTABILITY
+AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY SPECIAL, DIRECT,
+INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES WHATSOEVER RESULTING FROM
+LOSS OF USE, DATA OR PROFITS, WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR
+OTHER TORTIOUS ACTION, ARISING OUT OF OR IN CONNECTION WITH THE USE OR
+PERFORMANCE OF THIS SOFTWARE.
+***************************************************************************** */
+

```

Comment:
?

Reply:
It's coming from a dependency used by Leaflet, you have the same license block for Autocomplete: https://github.com/symfony/ux/blob/2.x/src/Autocomplete/assets/dist/controller.js

Reply:
See #1987